binding_generator.py

```diff
@@ -79,9 +85,11 @@ def generate_bindings(api_filepath, use_template_get_node, output_dir="."):
     shutil.rmtree(target_dir, ignore_errors=True)
     target_dir.mkdir(parents=True)
 
+    print("Built-in type config: " + double + "_" + bits)
```

Comment:
We really need to get rid of this `bits` thing and properly use `arch`, this it pretty confusing IMO.
@Faless has some WIP for it IINM, pending on reviewing/approving https://github.com/godotengine/godot/pull/55778

Reply:
@akien-mga Here's a PR for the godot-cpp repo for master: https://github.com/godotengine/godot-cpp/pull/759